tdp/cli/commands/browse.py

```diff
@@ -75,7 +77,9 @@ def process_deployments_query(session_class, limit, offset):
 
 
 def process_single_deployment_query(session_class, deployment_id):
-    deployment_headers = [key for key, _ in keyvalgen(DeploymentLog)]
+    deployment_headers = DeploymentLog.__table__.columns.keys() + [
+        str(DeploymentLog.services).split(".")[1]
+    ]
```

Comment:
Why ? The operations logs for the deployments will be queried anyway. (at l122)
You should keep it as it was.

```suggestion
    deployment_headers = [key for key, _ in keyvalgen(DeploymentLog)]
```